packages/app-builder/src/routes/transfercheck+/alerts+/inboxes._index.tsx

```diff
@@ -0,0 +1,11 @@
+import { serverServices } from '@app-builder/services/init.server';
```

Comment:
I'm perhaps nitpicking here but "inboxes" seems a bit weird considering half of it is rather an "outbox".
How about `alerts+/list` instead?